dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -103,8 +103,7 @@
 
 // Liberty Change: Ensures jaxws-2.2 matches similar performance with XLXP
 @NoJSR250Annotations
-public class JAXBDataBinding extends AbstractInterceptorProvidingDataBinding
-    implements WrapperCapableDatabinding, InterceptorProvider {
+public class JAXBDataBinding extends AbstractInterceptorProvidingDataBinding implements WrapperCapableDatabinding, InterceptorProvider {
```

Comment:
Can you undo the auto formatting changes to this file? It makes it very hard to spot what you've changed, but most importantly it makes it hard to spot differences between what we are changing from the original CXF code going forward. Keeping the original CXF format makes it easier for us to maintain the code overrides long term.